### PR TITLE
Bump to 2.0.3; CHANGELOG; SDK ceiling bump

### PR DESCRIPTION
**Pull request by srawlins**
I'd like to bump to 2.0.3 after #69 is merged.

### CODE REVIEW

**Comment by srawlins** [comment]
Update: #69 is merged.

**Comment by kevmoo** [comment]
While you're at it: rename `.analysis_options` to `analysis_options.yaml`?

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,9 +1,18 @@
+## 2.0.3
+
+* Add documentation for `when`, `verify`, `verifyNever`, `resetMockitoState`.
+* Expose `throwOnMissingStub`, `resetMockitoState`.
```
Comment (kevmoo) [inline CHANGELOG.md]: New features? This should be 2.1 then, right?

Comment (srawlins) [inline CHANGELOG.md]: Fair enough!

**Comment by srawlins** [comment]
Done!

**Pull request merged**